frontend/public/components/monitoring/alerting.tsx

```diff
@@ -589,9 +589,11 @@ const AlertMessage: React.FC<AlertMessageProps> = ({ alertText, labels, template
   });
 
   return (
-    <p>
-      <LinkifyExternal>{messageParts}</LinkifyExternal>
-    </p>
+    <div className="co-alert-manager__description">
```

Comment:
There does not appear to be a `block` named `co-alert-manager`, so a nested `element` (`description`) isn't following the rules of BEM.  Just a `block` rule seems more appropriate.